k8s/deployments/m3-deployment.yaml.example

```diff
@@ -77,7 +78,11 @@ spec:
             - name: MAIL_PASSWORD
               value: "<DEV_EMAIL_PASSWORD>"
             - name: APP_URL
-              value: localhost:3000
+              value: <APP_ADDRESS>
+            - name: S3_DOMAIN
+              value: <YOUR_DOMAIN>
+            - name: SETUP_USE_GLOBAL_BUCKETS
+              value: "true"
```

Comment:
Do we want this configuration to be enabled by default always?

Reply:
for now, yes